src/include/network/lan/complex.rb

```diff
@@ -219,8 +213,6 @@ def input_done?(ret)
 
       return Popup.ConfirmAbort(:painless) if Stage.initial
 
-      return ReallyAbort() if LanItems.GetModified
```

Comment:
isn't there any replacement of this behavior that we could use?

Reply:
fine tuned a bit